chall03/hopham.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+func httpGet(url string) string {
+	result, err := http.Get(url)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	content, err := ioutil.ReadAll(result.Body)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	result.Body.Close()
+	fmt.Println(string(content))
+	return (string(content))
+}
```

Comment:
This is pretty nice, but I would recommend avoid creating a function like this. It doesn't help to maintain your code if for example you are asked to also use a response header or response status code, you will have to tricks to use it.